x/gov/keeper/deposit.go

```diff
@@ -187,19 +187,31 @@ func (keeper Keeper) ChargeDeposit(ctx context.Context, proposalID uint64, destA
 
 	// burn the cancellation fee or sent the cancellation charges to destination address.
 	if !cancellationCharges.IsZero() {
-		// get the distribution module account address
-		distributionAddress := keeper.authKeeper.GetModuleAddress(disttypes.ModuleName)
+		// get the pool module account address
+		poolAddress := keeper.authKeeper.GetModuleAddress(pooltypes.ModuleName)
 		switch {
 		case destAddress == "":
 			// burn the cancellation charges from deposits
 			err := keeper.bankKeeper.BurnCoins(ctx, types.ModuleName, cancellationCharges)
 			if err != nil {
 				return err
 			}
-		case distributionAddress.String() == destAddress:
-			err := keeper.distrKeeper.FundCommunityPool(ctx, cancellationCharges, keeper.ModuleAccountAddress())
+		case poolAddress.String() == destAddress:
+			msg := pooltypes.NewMsgFundCommunityPool(cancellationCharges, keeper.ModuleAccountAddress().String())
```

Comment:
should we use the pulsar type to avoid the import here?

Reply:
I'm not sure how to do that, can you please explain

Reply:
pooltypes ""cosmossdk.io/api/cosmos/pool/v1" this useses the proto message type but avoids the import. It should be possible

Reply:
can you check if these changes makes sense